cmd/provider/main.go

```diff
@@ -39,6 +39,7 @@ func main() {
 		ansibleRolesPath       = app.Flag("ansible-roles-path", "Path where role(s) exists.").String()
 		syncPeriod             = app.Flag("sync", "Controller manager sync period such as 300ms, 1.5h, or 2h45m").Short('s').Default("1h").Duration()
 		pollInterval           = app.Flag("poll", "Poll interval controls how often an individual resource should be checked for drift.").Default("1m").Duration()
+		timeout                = app.Flag("timeout", "Controls how long Terraform processes may run before they are killed.").Default("20m").Duration()
```

Comment:
Controls how long `Ansible` processes may run before they are killed.

Reply:
oops, thanks for catching that, fixed